### PR TITLE
jormungandr: test realtime proxy on departures

### PR DESCRIPTION
**Pull request by TeXitoi**

### CODE REVIEW

File: source/jormungandr/tests/proxy_realtime_tests.py [hunk 1]
```diff
@@ -42,22 +42,34 @@ class MockedTestProxy(realtime_proxy.RealtimeProxy):
     def __init__(self, id):
         self.service_id = id
 
-    def next_passage_for_route_point(self, route_point):
+    @staticmethod
+    def _create_next_passages(passages):
```
Comment (xlqian) [inline source/jormungandr/tests/proxy_realtime_tests.py:46]: :+1:

File: source/jormungandr/tests/proxy_realtime_tests.py [hunk 2]
```diff
@@ -118,3 +130,18 @@ def test_stop_schedule_stop_ponit_has_no_rt(self):
         for dt in stop_schedule_B['date_times']:
             assert dt['data_freshness'] == 'base_schedule'
 
+    def test_stop_schedule(self):
+        query = 'stop_areas/S42/departures?from_datetime=20160102T1000&show_codes=true&count=7'
+        response = self.query_region(query, display=True)
+        departures = [(d['route']['name'], d['stop_date_time']['departure_date_time'])
+                      for d in response['departures']]
+        expected_departures = [
+            #("J", "20160102T100000"),
```
Comment (xlqian) [inline source/jormungandr/tests/proxy_realtime_tests.py]: why this code is commented?

File: source/jormungandr/tests/proxy_realtime_tests.py [hunk 3]
```diff
@@ -118,3 +130,18 @@ def test_stop_schedule_stop_ponit_has_no_rt(self):
         for dt in stop_schedule_B['date_times']:
             assert dt['data_freshness'] == 'base_schedule'
 
+    def test_stop_schedule(self):
```
Comment (antoine-de) [inline source/jormungandr/tests/proxy_realtime_tests.py]: I think you wanted to name this method test_departures

File: source/jormungandr/tests/proxy_realtime_tests.py [hunk 4]
```diff
@@ -118,3 +130,18 @@ def test_stop_schedule_stop_ponit_has_no_rt(self):
...
+        query = 'stop_areas/S42/departures?from_datetime=20160102T1000&show_codes=true&count=7'
+        response = self.query_region(query, display=True)
```
Comment (antoine-de) [inline source/jormungandr/tests/proxy_realtime_tests.py]: you should remove the display=True

File: source/jormungandr/tests/proxy_realtime_tests.py (hunk 2 above)
Comment (TeXitoi) [inline source/jormungandr/tests/proxy_realtime_tests.py]: Because empty RoutePoint is not working yet

**Pull request merged**